src/less/button/ds6/button-cta.less

```diff
@@ -16,61 +16,61 @@ a.cta-btn {
     }
 }
 
-a.cta-btn--p002-blue {
```

Comment:
This is a breaking change. We need to leave these old names in place for a minor version. We can remove them in 7.0.0.

Reply:
I actually thought of that _just_ as I was submitting the PR. Already in the works.